meinberlin/apps/kiezkasse/api.py

```diff
@@ -0,0 +1,58 @@
+from typing import List
+
+from django.db.models import QuerySet
+from rest_framework.filters import SearchFilter
+
+from adhocracy4.api.mixins import ModuleMixin
+from adhocracy4.modules.models import Module
+from meinberlin.apps.contrib.filters import NoExceptionFilterBackend
+from meinberlin.apps.contrib.filters import OrderingFilterWithDailyRandom
+from meinberlin.apps.ideas.filters import IdeaFilterSet
+
+from ..ideas.api import BaseIdeaViewSet
```

Comment:
maybe use the full import here and below, i.e. `meinberlin.apps.ideas`, `meinberlin.apps.kiezkasse.models`, etc.